llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -2441,6 +2441,9 @@ void ModuleAddressSanitizer::instrumentGlobals(IRBuilder<> &IRB, Module &M,
     // zero so we can copy the metadata over as is.
     NewGlobal->copyMetadata(G, 0);
 
+    // Attach "asan_instrumented" attribute to the new global.
+    NewGlobal->addAttribute(Attribute::SanitizeAddress);
```

Comment:
This would be replaced by "asan_instrumented" attributes, once #68865 gets approved and merged.